modules/terraform/workspace.go

```diff
@@ -0,0 +1,36 @@
+package terraform
+
+import (
+	"strings"
+	"testing"
+)
+
+// WorkspaceSelectOrNew runs terraform workspace with the given options and returns workspace name.
+// It tries to select a workspace with the given name, or it creates a new one if it doesn't exist.
+func WorkspaceSelectOrNew(t *testing.T, options *Options, name string) string {
```

Comment:
What is the return value?

Reply:
The return value is the name of the current workspace.

Reply:
Add that to the docs!